CalibTracker/SiStripHitEfficiency/plugins/SiStripHitEffFromCalibTree.cc

```diff
@@ -1004,6 +1034,22 @@ void SiStripHitEffFromCalibTree::makeHotColdMaps() {
 
 void SiStripHitEffFromCalibTree::makeTKMap(bool autoTagging = false) {
   LOGPRINT << "Entering TKMap generation!\n";
+
+  TTree* tree;
```

Comment:
so CLANG wants this to be initialized
https://cmssdt.cern.ch/SDT/jenkins-artifacts/pull-request-integration/PR-bf7a0b/32946/new-build-warnings.log